android/build.gradle

```diff
@@ -95,6 +96,31 @@ subprojects {
     boolean isSample = project.path.startsWith(":demos:") || project.path.startsWith(":tutorials:")
     boolean usesErrorProne = project.name in useErrorProneProjects
 
+    if (isAndroidLibrary) {
+        apply plugin: 'me.champeau.gradle.japicmp'
+
+        configurations {
+            baseline
+        }
+
+        dependencies {
+            baseline("com.uber.rib:${project.name}:${LAST_VERSION_NAME}") {
+                transitive = false
+                force = true
+            }
+        }
+
+        task japicmp(type: me.champeau.gradle.japicmp.JapicmpTask, dependsOn: 'assemble') {
+            oldClasspath = configurations.baseline.collect { zipTree(it) }.first().filter { it == 'classes.jar'}
+            newClasspath = zipTree(file("$buildDir/outputs/aar/${project.name}-release.aar")).filter { it == 'classes.jar'}
```

Comment:
ziptree looks neat, but couldn't get it to work w/ AARs on the older version of AGP so i had to stick w/ my unpackAar task